clang/include/clang/Basic/Attr.td

```diff
@@ -1309,12 +1309,12 @@ def SYCLType: InheritableAttr {
                             "specialization_id", "kernel_handler", "buffer_location",
                             "no_alias", "accessor_property_list", "group",
                             "private_memory", "aspect", "annotated_ptr", "annotated_arg",
-                            "stream", "sampler"],
+                            "compile_options", "stream", "sampler"],
```

Comment:
I don't see any special handling for this new `sycl_type` attribute value, why is it needed?

Reply:
Removed this and other unnecessary code. Thanks